src/permissions/PermissionController.test.ts

```diff
@@ -1123,19 +1120,19 @@ describe('PermissionController', () => {
         }),
       ).toThrow(new errors.UnrecognizedSubjectError('foo'));
     });
+  });
 
-    it('throws an error if the requested subject does not have the specified permission', () => {
+  describe('revokePermissions (idempotent tests)', () => {
+    // If permission is already revoked the method should not throw error
```

Comment:
Kind of the same question as before — can you explain how the permission is already revoked? I'm not really seeing this in the setup here. (It's also possible there's something I don't understand about the permissions controller, I admit I'm not the most familiar with it.)